### PR TITLE
Delete Ticket UI

### PR DESCRIPTION
**Pull request by ansonjwhe**
see #99

### CODE REVIEW

File: frontend/src/pages/Dashboard.js [hunk 1]
```diff
@@ -250,13 +274,26 @@ const Dashboard = () => {
         )
     }
 
+    const handleDeleteCurrentTicket = () => {
+        console.log('delete current ticket')
```
Comment (victorzheng02) [inline frontend/src/pages/Dashboard.js:278]: add a TODO: here, I'll handle it after this is merged

File: frontend/src/components/DeleteTicketButton.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+                    <AlertDialogFooter>
+                        <Button ref={cancelRef} onClick={onClose}>
+                            Cancel
+                        </Button>
+                        <Button
+                            colorScheme="red"
+                            onClick={() => {
+                                onClose()
+                                onDelete()
```
Comment (victorzheng02) [inline frontend/src/components/DeleteTicketButton.js:109]: you can add this now or I can add it later, but we should make this redirect to root as well as the ticket will be gone.

File: frontend/src/pages/Dashboard.js [hunk 3]
```diff
@@ -202,8 +214,20 @@ const Dashboard = () => {
...
-                        {`${currentTicket.type}-${currentTicket.id}: ${ticketData.name}`}
+                        {`${currentTicket.codename}`}
                     </Heading>
+                    <Flex flexDir="row" mb="12px">
+                        {/* Do not display delete button for WATO Cash */}
+                        {currentTicket.data.sf_link !== -1 && (
+                            <Button
+                                size="sm"
+                                colorScheme="red"
+                                onClick={onOpenDeleteTicket}
```
Comment (victorzheng02) [inline frontend/src/pages/Dashboard.js:225]: while making it pretty is not our immediate priority, I don't think there's harm in getting the layout right. I would move this to the right side of the codename heading

**Review by victorzheng02 — APPROVED** [review]
small comments, overall lgtm

File: frontend/src/components/DeleteTicketButton.js (hunk 2 above)
Comment (ansonjwhe) [reply frontend/src/components/DeleteTicketButton.js:109]: onDelete() should handle that logic

File: frontend/src/pages/Dashboard.js (hunk 3 above)
Comment (ansonjwhe) [reply frontend/src/pages/Dashboard.js:225]: this is what it looks like on JIRA

**Pull request merged**